stac_fastapi/pgstac/core.py

```diff
@@ -342,7 +342,10 @@ async def item_collection(
         bbox: Optional[BBox] = None,
         datetime: Optional[str] = None,
         limit: Optional[int] = None,
+        # Extensions
         token: Optional[str] = None,
+        filter_expr: Optional[str] = None,
+        filter_lang: Optional[str] = None,
```

Comment:
In theory we could add `sort`, `fields` and more but this is not officially supported by the stac spec

Reply:
ahhhh this wasn't in stac-fastapi so I assume it wasn't in the spec 🤦 

thanks @m-mohr

Reply:
Thanks for fixing this btw. Was just debugging why STAC Browser search queries weren't returning proper results and it seemed that there was a bug, but obviously it wasn't implemented. So good timing. I'd think supporting fields, sort and filter on the .../items endpoint would be a very welcome addition.

Reply:
> So good timing. I'd think supporting fields, sort and filter on the .../items endpoint would be a very welcome addition.

I'll try my best to add those but we might have issues downstream for add the conformance classes in the application which will require a new stac-fastapi release

Reply:
I see, thanks. I'm fine either way as long as the conformance classes are set correctly ;-)

Reply:
@m-mohr maybe you could help, there something I don't get about the conformance classes. if we have `https://api.stacspec.org/v1.0.0/ogcapi-features#sort` in the conformances, that doesn't mean the `/collections/{collectionId}/items` would have the `sort` parameter. It could mean that the `/search` endpoint support it. How do you deal with this?

Reply:
@vincentsarago No. For /search the conformance class is https://api.stacspec.org/v1.0.0/item-search#sort. For .../items it is https://api.stacspec.org/v1.0.0/ogcapi-features#sort though. So you can distinguish which endpoint supports sort. Similarly for filter etc.

Reply:
See
![grafik](https://github.com/user-attachments/assets/c4993668-6815-49c4-9957-c38eabef8b77)